### ISSUE TITLE
 Issues with Importing pnab Module in Jupyter Notebook and Accessing Online Version

### ISSUE THREAD

**Issue by zohar1021** [GT-NucleicAcids/pnab#2]
I have encountered issues with the pnab module in Jupyter Notebook, despite having successfully activated the pnab environment. When attempting to import and utilize pnab by executing the following code:
import pnab
pnab.builder()

I encounter the following **error message:**

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 1
----> 1 import pnab
      2 pnab.builder()

ModuleNotFoundError: No module named 'pnab'

It appears that the pnab module cannot be found, despite the environment being activated.

Additionally, when attempting to access the online version, I encounter **another issue.** 
---------------------------------------------------------------------------
Open Browser Console for more detailed log - Double click to close this message]
Failed to load model class 'ColormakerRegistryModel' from module 'nglview-js-widgets'
ReferenceError: s is not defined at Object. 

I would greatly appreciate assistance in resolving these issues. Thank you in advance for your help!

**Comment by alenaizan** [comment]
I think you need to install jupyter in the same conda environment:
```
conda create -n pnab -c conda-forge pnab jupyter
conda activate pnab
```
and use it.

For the online version, the link in the `launch binder` no longer works. You can use this one instead: https://mybinder.org/v2/gh/GT-NucleicAcids/pnab/HEAD?labpath=binder%2Fdriver.ipynb

I just noticed that for both the online version and the locally installed version, the output after running the examples is not displayed for some reason. However, you can navigate to the folder and see all the output PDB files and display them in another molecular viewer.

**Comment by alenaizan** [comment]
The issue has been resolved with the new version 1.1.6.
